bucket/winaero-tweaker.json

```diff
@@ -5,8 +5,11 @@
     "version": "0.14",
     "url": "https://winaero.com/winaerotweaker.zip",
     "hash": "a73ddf39cc92213106573f974ac032b396b6ff793a48156484416a1f45aa4a98",
+    "pre_install": [
+        "Rename-Item (Get-ChildItem $original_dir *.exe).FullName setup.exe"
```

Comment:
- Why original dir? 
- No quotes

Reply:
During an upgrade, *.exe would conflict with other .exe's in the "current" directory, whereas $original_dir is only going to contain the extracted files.

I can quote the constants to be strict about string usages. Actually I guess a lot of scoop scripts even add quotes when not necessary (eg, `\"$dir\" here https://github.com/lukesampson/scoop-extras/blob/master/bucket/edex-ui.json#L11`) -- is that just the expected style? `$dir` is just a powershell variable (as opposed to something being string-replaced prior to execution), isn't it?

Reply:
For more info: https://stackoverflow.com/questions/5571579/escaping-splatted-variables-in-powershell-invoke-expression

TLDR: when using `Invoke-Expression` powershell is "replacing" variables with pure value, so commands like `Get-Child item $dir` will become `Get-ChildItem C:\Whatever with spaces\cosi`, which ends with error.

Reply:
I am still not sure about that $original_dir. Will test when arrive home, but i think normal $dir should be enough for everything. /cc @r15ch13

Reply:
I guess if $dir (current) is pointed at the currently-being-installed directory by then it would be fine? That requires knowing more about the internals of scoop than I currently know (which, IMHO, should be unnecessary for building a mainfest file). It also isn't clear to me what benefit `$dir` would have over `$original_dir`?

I personally think `$original_dir` (the `0.14` dir)  makes more sense semantically, if not technically, than `$dir` (`current`) but don't have a strong enough opinion either way.

Reply:
`$dir` is used consistently in manifests.
`$dir` contains the versioned path at the time the installer gets executed and gets changed later to `current`

https://github.com/lukesampson/scoop/blob/3830d9f5f2cb544c9758c5ef3b591c9b0255df6a/lib/install.ps1#L39-L54